### DISCUSSION TITLE
highlighting and commenting chunk of text

### DISCUSSION THREAD

**Discussion by quazardous** [zim-desktop-wiki/zim-desktop-wiki#2001] [Ideas]
I need something to highlight parts of a wiki page.

My use case is flagging/highlighting changes from an original script or add temporary notes (I use zim as my star wars D6 notebook).

EDIT: this feature is like the commenting feature of GoogleDoc.

**Comment by introt** [comment]
You can highlight with Ctrl+U and link the same text to other pages which contain your notes. I'd also use anchors/links to headings to keep all the notes on the same page, and then open the script in a new window so that you can just click the links and have the main window scroll to the appropriate position.

There's an open ticket for colors: #268

**Comment by quazardous** [comment]
It could be an interesting feature to wrap the whole process of commenting/highlighting a chunk of text. I use this a lot with GoogleDoc.

Highlight is more semantic than decoration for me (it's part of the writing process). So easily adding a comment or a link to a chunk of text could be great.

Maybe it could be merged with the task feature.

The process:

- select a chunk of text
- hit CTRL-something or a button
- enter an optional comment in the confirmation popup (or something else than a popup)
- a comment/task is created

The rendering could be inline, popinfo or in the task panel only.

**Reply by introt** [reply]
Please add this to #1807, thanks!

**Reply by quazardous** [reply]
will do thx
